### PR TITLE
[CloudPC]Add disk encryption type to CloudPC entity

### PR DESCRIPTION
**Pull request by BenxiangXiaMSFT**

### CODE REVIEW

**Comment by RuiHou105** [comment]
Roadmap work item number:14978

**Review by RuiHou105 — APPROVED** [review]

File: api-reference/beta/api/cloudpc-get.md [hunk 1]
```diff
@@ -155,7 +155,8 @@ Content-Type: application/json
     },
     "userPrincipalName": "pmitchell@cpccustomer001.onmicrosoft.com",
     "lastModifiedDateTime": "2020-11-03T18:14:34Z",
-    "gracePeriodEndDateTime": "2020-11-010T20:00:34Z"
+    "gracePeriodEndDateTime": "2020-11-010T20:00:34Z",
+    "diskEncryptionState": "notAvailable"
```
Comment (AshleyYangSZ) [inline api-reference/beta/api/cloudpc-get.md:159]: Will this be returned by default? If so, please update list cloudpcs API example in this PR as well. Thanks a lot!

Comment (BenxiangXiaMSFT) [reply api-reference/beta/api/cloudpc-get.md:159]: Examples updated.

**Comment by Danielabom** [comment]
I contacted @BenxiangXiaMSFT, but it seems that they are not in. Today, I reached out to @RuiHou105 and @AshleyYangSZ.

File: api-reference/beta/resources/cloudpc.md [hunk 2]
```diff
@@ -47,6 +47,7 @@ Represents a cloud-managed virtual desktop. This Cloud PC is also enrolled into
 |:---|:---|:---|
 |aadDeviceId|String|The Azure Active Directory (Azure AD) device ID of the Cloud PC.|
 |connectivityResult|[cloudPcConnectivityResult](../resources/cloudpcconnectivityresult.md)|The connectivity health check result of a Cloud PC, including the updated timestamp and whether the Cloud PC is able to be connected or not.|
+|diskEncryptionState|[cloudPcDiskEncryptionState](#cloudpcdiskencryptionstate-values)|The disk encryption applied to the Cloud PC. Possible values: `notAvailable`, `notEncryped`, `encryptedUsingPlatformManagedKey`, `encryptedUsingCustomerManagedKey`.|
```
Comment (Danielabom) [inline api-reference/beta/resources/cloudpc.md:50]:
```suggestion
|diskEncryptionState|[cloudPcDiskEncryptionState](#cloudpcdiskencryptionstate-values)|The disk encryption applied to the Cloud PC. Possible values: `notAvailable`, `notEncryped`, `encryptedUsingPlatformManagedKey`, `encryptedUsingCustomerManagedKey`, and `unknownFutureValue`.|
```

File: api-reference/beta/resources/cloudpc.md [hunk 3]
```diff
@@ -68,6 +69,16 @@ Represents a cloud-managed virtual desktop. This Cloud PC is also enrolled into
 |userAccountType|[cloudPcUserAccountType](../resources/cloudpcorganizationsettings.md#cloudpcuseraccounttype-values)|The account type of the user on provisioned Cloud PCs. Possible values are: `standardUser`, `administrator`, and `unknownFutureValue`.|
 |userPrincipalName|String|The user principal name (UPN) of the user assigned to the Cloud PC.|
 
+### cloudPcDiskEncryptionState values
+
+|Member|Description|
+|:---|:---|
+|notAvailable|The Cloud PC is not provisioned, or in a state that encryption is not available.|
```
Comment (Danielabom) [inline api-reference/beta/resources/cloudpc.md:76]:
```suggestion
|notAvailable|The Cloud PC is not provisioned, or in a state where encryption is not available.|
```

File: api-reference/beta/resources/cloudpc.md [hunk 4]
```diff
@@ -68,6 +69,16 @@ Represents a cloud-managed virtual desktop. This Cloud PC is also enrolled into
...
+|notEncryped|The Cloud PC should be encryped, but the encryption is not done yet (reserved, should not happen).|
```
Comment (Danielabom) [inline api-reference/beta/resources/cloudpc.md:77]:
```suggestion
|notEncryped|The Cloud PC should be encrypted, but the encryption is not done yet (reserved, should not happen).|
```

File: api-reference/beta/resources/cloudpc.md [hunk 5]
```diff
@@ -68,6 +69,16 @@ Represents a cloud-managed virtual desktop. This Cloud PC is also enrolled into
...
+|encryptedUsingPlatformManagedKey|The Cloud PC is encrypted using platform managed key. This is the default if customer managed key is not enabled.|
```
Comment (Danielabom) [inline api-reference/beta/resources/cloudpc.md:78]:
```suggestion
|encryptedUsingPlatformManagedKey|The Cloud PC is encrypted using platform managed key. This is the default value if the customer-managed key is not enabled.|
```

File: api-reference/beta/resources/cloudpc.md [hunk 6]
```diff
@@ -68,6 +69,16 @@ Represents a cloud-managed virtual desktop. This Cloud PC is also enrolled into
...
+|encryptedUsingCustomerManagedKey|The Cloud PC is encrypted using customer managed key.|
```
Comment (Danielabom) [inline api-reference/beta/resources/cloudpc.md:79]:
```suggestion
|encryptedUsingCustomerManagedKey|The Cloud PC is encrypted using customer-managed key.|
```

**Review by Danielabom — CHANGES_REQUESTED** [review]
Hi @BenxiangXiaMSFT, based on our conversation, the word encrypted appears misspelled several times in the PR. Please confirm whether you can address the changes in this PR or if you'll create a new one. Thanks

**Review by AshleyYangSZ — CHANGES_REQUESTED** [review]
@BenxiangXiaMSFT Please help update the PR according to the comments!

File: api-reference/beta/resources/cloudpc.md (hunk 2 above)
Comment (AshleyYangSZ) [reply api-reference/beta/resources/cloudpc.md:50]: @BenxiangXiaMSFT Please help resolve this comment asap.

**Review by Danielabom — APPROVED** [review]
Changes look good . PR approved. Thanks, @BenxiangXiaMSFT.

**Pull request merged**